File: idk1.js
let nilai = 100;
switch(true){
    case (nilai >= 90):
        console.log('Nilai A');
        break;
    case (nilai >= 80 && nilai <= 89):
        console.log('Nilai B');
        break;
    case (nilai >= 70 && nilai <= 79):
        console.log('Nilai C');
        break;
    case (nilai >= 60 && nilai <= 69):
        console.log('Nilai D');
        break;
    default:
        console.log('Nilai E');
}

File: idk1.py
nilai2 = input("Masukkan nilai: ")
nilai = int(nilai2)
match nilai:
    case n if n >= 90:
        print('Nilai A')
    case n if n >= 80 and n <= 89:
        print('Nilai B')
    case n if n >= 70 and n <= 79:
        print('Nilai C')
    case n if n >= 60 and n <= 69:
        print('Nilai D')
    case n if n < 60:
        print('Nilai E')